qemu/tests/cfg/driver_load_stress.cfg

```diff
@@ -0,0 +1,55 @@
+- driver_load_stress:
+    type = driver_in_use
+    start_vm = yes
+    kill_vm_on_error = yes
+    check_guest_bsod = yes
+    kill_vm_on_error = yes
+    login_timeout = 240
+    suppress_exception = False
+    driver_load_cmd = C:\devcon.exe enable @DRIVER_ID
+    driver_unload_cmd = C:\devcon.exe disable @DRIVER_ID
+    driver_id_cmd = C:\devcon.exe find * | find "VirtIO"
+    only Windows
+    variants:
+        - before_bg_test:
+            run_bg_flag = "before_bg_test"
+        - during_bg_test:
+            run_bg_flag = "during_bg_test"
+        - after_bg_test:
+            run_bg_flag = "after_bg_test"
+    variants:
+        - with_block:
+            sub_test = driver_load
+            driver_name = viostor
+            run_bgstress = iozone_windows
+            drive_format_image1 = ide
+            cdrom_cd1 = isos/windows/winutils.iso
+            images += " stg"
+            image_name_stg = "images/storage"
+            image_size_stg = 4G
+            drive_format_stg = virtio
+            force_create_image_stg = yes
+            remove_image_stg = yes
+            iozone_cmd = "%s:\Iozone\iozone.exe -azR -r 64k -n 1G -g 4G -M -b iozone.xls -f %s:\testfile"
+            iozone_timeout = 7200
+            wait_bg_time = 180
+            driver_id_pattern = "(.*?):.*?VirtIO SCSI Disk Device"
+            disk_letter = I
```

Comment:
For the disk_letter is fixed, is there another way for the problem?
It's a little wired.